packages/activemodel-adapter/tests/integration/active-model-serializer-namespaced-modelname-new-test.js

```diff
@@ -0,0 +1,155 @@
+var SuperVillain, EvilMinion, YellowMinion, DoomsdayDevice, MediocreVillain, TestSerializer, env;
```

Comment:
not sure, but maybe a missing dash in the file name ? `active-model-serializer-namespaced-model*-*name-new-test.js`?